README.md

```diff
@@ -45,12 +44,6 @@ To build a VM machine from this repo's source, see the [instructions](docs/creat
 [macOS-10.15]: https://github.com/actions/runner-images/blob/main/images/macos/macos-10.15-Readme.md
 [self-hosted runners]: https://help.github.com/en/actions/hosting-your-own-runners
 
-## Ongoing migrations
-
-There are currently two migrations in progress:

```

Comment:
Let's change this line to 
```
There are currently no migrations.
```
and delete the lines below.

We should also add a doc (can be outside of this PR) that explains migrations and what users should expect during these time periods.

Reply:
done